UpdateRCConfiguration/function.json

```diff
@@ -0,0 +1,20 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
```

Comment:
why `anonymous`? unless there are specific reason, we should always use `function` as default `authLevel`

(with `function` you have to provide an api key to use the endpoint)

Reply:
These functions are anonymous because they are deployed to AKS and are not receiving the x-functions-key header. We check permission on APIM then redirect to the AKS cluster.

Reply:
why are we using `AKS` for these az functions?

Reply:
In few words: it's a previous team's legacy.
These functions, were born and deployed to AKS long time ago. I have the desire to migrate them to plain Azure Functions, but we have no capacity for this at the moment.

Reply:
ok